pagopa/io-app
# fix: [IOBP-462] Missing transaction details data

## Short description
fix of a small bug which didn't make a part of the screen render
<img src="https://github.com/pagopa/io-app/assets/60693085/e7920c0d-8d62-474c-9438-97e298e042da" width=200/>

## List of changes proposed in this pull request
- removed dead code
- small logic fix
- addition of I18n keys for when there is no PSP name
- small refactors

## How to test
with the `dev-server` active, navigate to wallet, then down to the transaction history.
done that, open up a transaction's details and make sure that
a. the screen is correctly rendered
b. there is a "fee" info in the top part as in the screenshot (in case it does not render, try opening another op's details, since it is expected to be *possibly* missing sometimes
c. the screen correctly displays in a loading state when loading
